rules/order.js

```diff
@@ -15,13 +18,35 @@ module.exports = {
         'right',
         'bottom',
         'left',
+        'visibility',
+        'opacity',
       ],
     },
     {
       order: 'strict',
       properties: [
         'transform',
         'transform-origin',
+        // Animating transforms is common, so let's group them together
```

Comment:
IMO this actually isn't really true, we often also transition opacity and, less frequently, background-color. Always has felt right to me for these to be at the bottom.